.github/workflows/hash-backend-cd.yml

```diff
@@ -5,18 +5,15 @@ on:
     branches:
       - main
     paths:
+      - "apps/hash-graph/**"
```

Comment:
Do we want to sort them by their name?

Reply:
Probably yes, but I started out just grouping them by their path at least